protocol/common/raw_messages_persistence.go

```diff
@@ -405,10 +410,15 @@ func (db *RawMessagesPersistence) CompleteMessageSegments(hash []byte, sigPubKey
 		return err
 	}
 
-	_, err = tx.Exec("INSERT INTO message_segments_completed (hash, sig_pub_key) VALUES (?,?)", hash, sigPubKeyBlob)
+	_, err = tx.Exec("INSERT INTO message_segments_completed (hash, sig_pub_key, timestamp) VALUES (?,?,?)", hash, sigPubKeyBlob, timestamp)
 	if err != nil {
 		return err
 	}
 
 	return err
 }
+
+func (db *RawMessagesPersistence) RemoveMessageSegmentsCompletedOlderThan(timestamp int64) error {
+	_, err := db.db.Exec("DELETE FROM message_segments_completed WHERE timestamp < ?", timestamp)
```

Comment:
is there an index on timestamp?

Reply:
it is now -> https://github.com/status-im/status-go/compare/4a75ec896b03a9c6dee7c1ae997f857ba66dd05f..146c4dbb478a4ff2fb7da34433ccc3af4e2ae99e